src/pdf/dccCoverPage.js

```diff
@@ -200,7 +200,7 @@ function proofsList(doc, proofs) {
             );
         }
         if (isRecovery(proof)) {
-            return t(doc.local, "cover.recoveryProof");
+            return t(doc.locale, "cover.recoveryProof");
         }
```

Comment:
i see a change from ```doc.local``` to ```doc.locale```
but there is still a ```doc.local```
or is this just a confusion because locale is languange and local is LOCAL ?

Reply:
Nope, the other one is a bug, too...